app/__init__.py

```diff
@@ -1,5 +1,7 @@
 from flask import Flask
 from .db import db, migrate
+from .routes.task_routes import tasks_bp
+from .routes.goal_routes import goals_bp 
```

Comment:
We prefer to name the `task_bp` and `goal_bp` just `bp` in their respective route files, and then either do an `import as` here to differentiate them, or import each entire route module and access its Blueprint using dot notation.

```suggestion
from .routes.task_routes import bp as tasks_bp
from .routes.goal_routes import bp as goals_bp 
```